sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
This addition intended to bring the namespace in sync with the API documentation, which lists these.
If the API documentation is wrong, they should be removed there (and here).

Agreed that it's unrelated to C removal; it's just something that came up as I was looking at the API docs, and I didn't consider it worth doing an extra PR for that. YMMV.